pkg/prediction/config/types.go

```diff
@@ -10,7 +10,20 @@ type AlgorithmModelConfig struct {
 	UpdateInterval time.Duration
 }
 
+type ModelInitMode string
+
+const (
+	// means recover or init the algorithm model directly from history datasource, this process may block because it is time consuming for data fetching & model gen
```

Comment:
Can we make sure which  mode is default?

Reply:
this param is specified by caller or user, if it is not specified, default is history mode, original logic